examples/pyo3-pytests/tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+# can't install from sdist because local pyo3 repo can't be included in the sdist
+skipsdist = true
+
+[testenv]
+description = Run the unit tests under {basepython}
+deps = -rrequirements-dev.txt
+commands =
+    # Use pip master with in-tree-build feature (to be released in pip 21.0)
```

Comment:
21.0.1 is released, does this work with pip from pypi?

Reply:
Outdated comment sadly - the branch missed the merge window so will be released in `21.1`.

Reply:
```suggestion
    # Use pip master with in-tree-build feature (to be released in pip 21.1)
```